src/Symfony/Component/Config/Definition/Builder/ExprBuilder.php

```diff
@@ -63,9 +63,7 @@ public function always(\Closure $then = null): static
      */
     public function ifTrue(\Closure $closure = null): static
     {
-        if (null === $closure) {
-            $closure = static function ($v) { return true === $v; };
-        }
+        $closure ??= static function ($v) { return true === $v; };
```

Comment:
Maybe we should browse the codebase for potential arrow functions as well. 🤔 

```suggestion
        $closure ??= static fn ($v) => true === $v;
```

But this should probably be a separate PR.